dsb-integration/src/test/java/com/spectralogic/dsbrowser/integration/tasks/Ds3PutJobTest.java

```diff
@@ -146,7 +146,7 @@ public void executeJob() throws InterruptedException {
         Platform.runLater(() -> {
             jobWorkers.execute(ds3PutJob);
             ds3PutJob.setOnSucceeded(event -> {
-                if (!ds3PutJob.isJobFailed()) {
+                if (!Ds3JobTask.isJobFailed()) {
```

Comment:
I'm confused by this.  Why was it changed to call the static form on the parent class?